superagentx_handlers/best_buy.py

```diff
@@ -0,0 +1,63 @@
+from abc import ABC
+import aiohttp
+from superagentx.handler.base import BaseHandler
+from superagentx.handler.decorators import tool
+
+base_url = "https://api.bestbuy.com/v1/products"
+show_options = ("show=customerReviewAverage"
+                ",customerReviewCount"
+                ",dollarSavings"
+                ",image"
+                ",includedItemList.includedItem"
+                ",modelNumber"
+                ",name"
+                ",onlineAvailability"
+                ",onSale"
+                ",percentSavings"
+                ",regularPrice"
+                ",salePrice"
+                ",sku"
+                ",thumbnailImage")
+default_pagination = "&pageSize=100"
+response_format = "&format=json"
+
+
+class BestbuyHandler(BaseHandler, ABC):
+    """
+        A handler for interacting with the Best Buy API. This class provides methods
+        to retrieve information about products from Best Buy's inventory.
+
+        Attributes:
+            api_key (str): The API key for authenticating requests to the Best Buy API.
+
+        Methods:
+            get_best_buy_info(keyword: str, pagination: str = None) -> dict:
+                Retrieves product information from the Best Buy API based on the specified keyword.
+    """
+
+    def __init__(self,
+                 *,
+                 api_key: str
+                 ):
+        super().__init__()
+        self.api_key = api_key
+
+    @tool
+    async def get_best_buy_info(self, keyword: str, pagination: str = None):
```

Comment:
Align the args. And rename `keyword` argument to some other valid name

Reply:
Aligned and name changed instead keyword to "search_text"